openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx

```diff
@@ -604,7 +603,7 @@ export const createNodes = (
       className: '',
       data: {
         node,
-        isRootNode: getDecodedFqn(entityFqn) === node.fullyQualifiedName,
+        isRootNode: entityFqn === node.fullyQualifiedName,
```

Comment:
Don't we need to decode the fqn here? can explain why?

Reply:
We don't need to decode here, as we are already passing decodedFqn as an arguments.

Reply:
Thanks for confirming @Ashish8689